app/controllers/api/v1/books_controller.rb

```diff
@@ -1,13 +1,30 @@
 module Api
   module V1
     class BooksController < ApiController
+      skip_before_action :authenticate_request, :current_user, only: [:isbn]
+
       def index
         render json: Book.all
       end
 
       def show
         render json: Book.find(params[:id]), serializer: FullBookSerializer
       end
+
+      def isbn
+        book_by_isbn = Book.find_by(isbn: isbn_param['isbn'])
```

Comment:
It's ok for now, but we will not store books in our system anymore, the info will be taken from an external API